src/components/DataTable/table-cells/TableHeadCell.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {render, screen, fireEvent} from '@testing-library/react';
```

Comment:
As other tests, we use [userEvent](https://testing-library.com/docs/user-event/intro) instead of `fireEvent`